exercises/practice/collatz-conjecture/collatz-conjecture.test.ts

```diff
@@ -22,16 +22,23 @@ describe('CollatzConjecture', () => {
   })
 
   xit('zero is an error', () => {
-    const expected = 'Only positive numbers are allowed'
+    const expected = 'Only positive integers are allowed'
     expect(() => {
       steps(0)
     }).toThrowError(expected)
   })
 
   xit('negative value is an error', () => {
-    const expected = 'Only positive numbers are allowed'
+    const expected = 'Only positive integers are allowed'
     expect(() => {
       steps(-15)
     }).toThrowError(expected)
   })
+
+  xit('negative value is an error', () => {
```

Comment:
This is not a negative value that's being tested.

Reply:
Corrected, thanks!